modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -60,9 +64,31 @@ const RangeUtils = (dom: DOMUtils): RangeUtils => {
     );
   };
 
+  /**
+   * Expands the range of the selection to contain the entire when nothing is selected
```

Comment:
Not sure how to describe this right now it's just doing one thing in the future it might be doing more things. But maybe this is more accurate to what it's doing right now. It will not expand when the range is at the start or end of the word.

```suggestion
   * Expands the range of the selection to contain the entire word when the selection is collapsed within the word
```